rococo-parachains/src/cli.rs

```diff
@@ -88,24 +88,6 @@ pub struct ExportGenesisWasmCommand {
 	pub chain: Option<String>,
 }
 
-#[derive(Debug, StructOpt)]
-pub struct RunCmd {
-	#[structopt(flatten)]
-	pub base: sc_cli::RunCmd,
-
-	/// Id of the parachain this collator collates for.
-	#[structopt(long)]
-	pub parachain_id: Option<u32>,
-}
-
-impl std::ops::Deref for RunCmd {
-	type Target = sc_cli::RunCmd;
-
-	fn deref(&self) -> &Self::Target {
-		&self.base
-	}
-}
```

Comment:
See how the previous implementation circumvent the need of calling the "base" `RunCmd` by using `Deref`. Unfortunately you can't do the same because you want to override the `fn role()`.

Reply:
Ohh, I wondered what the point of this `Deref` was. I see how it was useful now.

Reply:
It was a bit lazy :sweat: